### DISCUSSION TITLE
footnotes references at end of filename

### DISCUSSION THREAD

**Discussion by danymat** [srid/emanote#287] [Ideas]
Hello ! Thanks for this wonderful app, I'm using it in https://notes.danielmathiot.tech !

I was using neuron before, and one thing I miss from it is the ability to have footnotes references at the end of filename, like this md file:

```md
# 202107070345 Définir la portée du test[^1]
#pci2017

La portée du test doit inclure:

- Localisations des données personnelles
- les applications qui gèrent transmettent et enregistrent les données personnelles.
- Les points d'accès

[^1]:PCI Penetration Testing Guidance 2017 Chapter 2.2
```

That currently translates into: https://notes.danielmathiot.tech/202107070345%20Définir%20la%20portée%20du%20test.html

**Comment by srid** [comment]
@danymat Ya, this is a limitation of Emanote where note titles (which are determined from level 1 headings or note filename if one doesn't exist) are handled _separately_ from the document. I have not seen anyone use footnotes in level 1 headings. As a workaround you can move it to the body, eg:

```markdown
# 202107070345 Définir la portée du test
#pci2017

La portée du test[^1] doit inclure:
...
```
